src/grt/test/gcd.ok

```diff
@@ -64,8 +64,10 @@ metal8            4512             0            0.00%             0 /  0 /  0
 metal9            2162             0            0.00%             0 /  0 /  0
 metal10           2209             0            0.00%             0 /  0 /  0
 ---------------------------------------------------------------------------------------
-Total           149816          3343            2.23%             0 /  0 /  0
+Total           148811          3343            2.25%             0 /  0 /  0
 
 [INFO GRT-0018] Total wirelength: 10258 um
 [INFO GRT-0014] Routed nets: 563
-No differences found.
+Differences found at line 914.
+105000 67200 113400 71400 metal1
+105000 67200 109200 71400 metal1
```

Comment:
There should be no differences in the .ok files

Reply:
Ouch, let me fix that